games-arcade/rocksndiamonds/patches/rocksndiamonds-3.3.1.2.patchset

```diff
@@ -30,10 +30,10 @@ index a67cd14..8493bb9 100644
  
  SRCS =	main.c		\
 -- 
-1.8.3.4
+2.7.0
```

Comment:
If you want to have the smallest git diff, you can replace "2.7.0" by "1.8.2.4".